src/service/poc_lora.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package helium.poc_lora;
+
+message poc_lora_beacon_req_v1 {
+  bytes poc_key = 1;
+  bytes pub_key = 2;  
+  bytes entropy = 3;
+  bytes data = 4;
+  uint64 timestamp = 5;
```

Comment:
Several of these fields are going to have unknown values because this is a *transmit* claim, not a receive claim. Specifically timestamp (unless we can find a way to send a timed transmission without it being a packet response), signal and SNR are all unknown to the sender.

Reply:
addressed here: https://github.com/helium/proto/pull/154/commits/69497c85fbe35ea4f5ba6d79155d9629b21854b8